src/main/java/io/cdap/WholeFileSource.java

```diff
@@ -14,23 +14,23 @@
  * the License.
  */
 
-package co.cask;
-
-import co.cask.cdap.api.annotation.Description;
-import co.cask.cdap.api.annotation.Macro;
-import co.cask.cdap.api.annotation.Name;
-import co.cask.cdap.api.annotation.Plugin;
-import co.cask.cdap.api.data.batch.Input;
-import co.cask.cdap.api.data.batch.InputFormatProvider;
-import co.cask.cdap.api.data.format.StructuredRecord;
-import co.cask.cdap.api.data.schema.Schema;
-import co.cask.cdap.api.dataset.lib.KeyValue;
-import co.cask.cdap.api.plugin.PluginConfig;
-import co.cask.cdap.etl.api.Emitter;
-import co.cask.cdap.etl.api.PipelineConfigurer;
-import co.cask.cdap.etl.api.batch.BatchSource;
-import co.cask.cdap.etl.api.batch.BatchSourceContext;
-import co.cask.format.WholeFileInputFormat;
+package io.cdap;
```

Comment:
We change `co.cask.hydrator` to `io.cdap.plugin`. Is the change here specific to this case because it is a plugin? I am not sure if this is consistent across all the plugins hence the question.

Reply:
if we're not doing it everywhere perhaps its ok